backend/FwLite/MiniLcm/InMemoryApi.cs

```diff
@@ -191,6 +197,16 @@ public async Task CreatePartOfSpeech(PartOfSpeech partOfSpeech)
         throw new NotImplementedException();
```

Comment:
What is the InMemoryApi used for? Having (Create/Update/Delete)PartOfSpeech be unimplemented didn't throw any errors during unit testing. Do I need to implement these methods?

Reply:
I don't think it's actually being used anymore. @myieye what do you think? should we just delete it? It could be considered a reference implementation, but I don't really want to support that, I'd rather say either Crdt or FW is our reference.